src/sordid-dragon.jquery.coffee

```diff
@@ -61,20 +61,32 @@ do ($=jQuery) ->
     # create a ghost under the cursor for us. That doesn't happen with touch
     # events, so we create one ourselves.
     $ghost = null
+    $ghostContainer = $("<div />")
+      .addClass("ghost-container")
+      .css
+        display: "none"
+        position: "fixed"
+        zIndex: 1000
     showGhost = (pageY) ->
+      $ghostContainer.appendTo("body").show()
+
       unless $ghost.is(":visible")
-        $ghost.addClass "sordidDragon-ghost"
-        $ghost.css
-          position: "fixed"
-          opacity: 1
-        $parent.append $ghost
+        $ghost
+          .addClass "sordidDragon-ghost"
+          .css
+            position: "fixed"
+            opacity: 1
+        $ghostContainer.append $ghost
+
       $ghost.css
         left: $placeholder.offset().left
         top: (pageY - ($placeholder.outerHeight() / 2)) - window.scrollY
         width: $placeholder.outerWidth() - window.scrollX
 
     hideGhost = ->
       $ghost?.remove()
+      $ghostContainer?.html("")
```

Comment:
`$ghost` I believe is different every time, it's the element that you're dragging to a new position. We can't show and hide it–we can show and hide it; but will need to change the text, value, and or anything else about that element to match the *selected* or dragging element.

I think the original authors thought it was easier to just clone it and remove it.